### PR TITLE
[chng] Update expected file count

### LINKED ISSUE #1440: chng failing with `assert len(filepaths_to_download) <= 6, "more files dropped than expected" AssertionError: more files dropped than expected`
**Actual Behavior:**

source: `chng`

Per @jingjtang's investigation:

>It seems that there are no counts delivered from 2021-12-23 to 2021-12-26.

Fails with:

Slack ref: https://delphi-org.slack.com/archives/C01LZ3A2UMU/p1640748872124100

<details>
  <summary>:exclamation: Indicator exception</summary>
  
  ```
  Logger: delphi_changehc.run
Timestamp: 2021-12-29T03:34:02.771Z
Exception: Traceback (most recent call last):
 File "/home/indicators/.pyenv/versions/3.8.2/lib/python3.8/runpy.py", line 193, in _run_module_as_main
   return _run_code(code, main_globals, None,
 File "/home/indicators/.pyenv/versions/3.8.2/lib/python3.8/runpy.py", line 86, in _run_code
   exec(code, run_globals)
 File "/mnt/data2/indicators/runtime/changehc/env/lib/python3.8/site-packages/delphi_utils/runner.py", line 64, in <module>
   run_indicator_pipeline(indicator_module.run.run_module,
 File "/mnt/data2/indicators/runtime/changehc/env/lib/python3.8/site-packages/delphi_utils/runner.py", line 43, in run_indicator_pipeline
   indicator_fn(params)
 File "/mnt/data2/indicators/runtime/changehc/delphi_changehc/run.py", line 129, in run_module
   file_dict = retrieve_files(params, filedate, logger)
 File "/mnt/data2/indicators/runtime/changehc/delphi_changehc/run.py", line 29, in retrieve_files
   download_counts(filedate, params["indicator"]["input_cache_dir"], params["indicator"]["ftp_conn"])
 File "/mnt/data2/indicators/runtime/changehc/delphi_changehc/download_ftp_files.py", line 64, in download_counts
   get_files_from_dir(sftp, filedate, out_path)
 File "/mnt/data2/indicators/runtime/changehc/delphi_changehc/download_ftp_files.py", line 36, in get_files_from_dir
   assert len(filepaths_to_download) <= 6, "more files dropped than expected"
AssertionError: more files dropped than expected
```
</details>

**Expected behavior**

`chng` should not fail.

**Context**

This looks like provider issue. As this is happening during the Winter Holiday period we aren't meeting with them for a week or two.

@jingjtang has emailed CHNG 2022-01-03, waiting for CHNG to add those files to the SFTP server

### PR DESCRIPTION
**Pull request by krivard**
### Description
We now get an additional file in the drop for flu inpatient.

### Changelog
Itemize code/test/documentation changes and files added/removed.
- download_ftp_files.py: update expected file count

### Fixes 
- Fixes #1440

### CODE REVIEW

**Comment by jingjtang** [comment]
@krivard Was the inpatient Flu data added for all the dates before 2021-12-27 at once? The first inpatient Flu data that I can see is for 2021-12-22 which is not 2021-12-27 or 2021-12-28(the problematic date).

**Comment by krivard** [comment]
The new files were added one at a time during the normal upload cycle (no backfill).

The first inpatient flu file was uploaded on 2021-12-23 at 5pm, and the CHNG pipeline would have run at 10:30pm that day. [According to the logs](https://logs-prod-01.delphi.cmu.edu/app/dashboards#/view/16016d70-574e-11eb-88e3-e914236be05f?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-30d%2Fd,to:now))), 2021-12-23 is the first day that the exception in #1440 occurred, and it occurred each day after that as well (including the 27th and 28th).

The most recent CHNG issue that we have in covidcast is 20211222.

I'm not sure whether that answers your question -- was there something else I'm missing?

**Comment by krivard** [comment]
@jingjtang Updated implementation & fixed tests

**Comment by jingjtang** [comment]
> The new files were added one at a time during the normal upload cycle (no backfill).
> 
> The first inpatient flu file was uploaded on 2021-12-23 at 5pm, and the CHNG pipeline would have run at 10:30pm that day. [According to the logs](https://logs-prod-01.delphi.cmu.edu/app/dashboards#/view/16016d70-574e-11eb-88e3-e914236be05f?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-30d%2Fd,to:now))), 2021-12-23 is the first day that the exception in #1440 occurred, and it occurred each day after that as well (including the 27th and 28th).
> 
> The most recent CHNG issue that we have in covidcast is 20211222.
> 
> I'm not sure whether that answers your question -- was there something else I'm missing?

Yeah, that makes sense. Thanks!

File: changehc/tests/test_download_ftp_files.py [hunk 1]
```diff
@@ -51,22 +52,18 @@ def test_get_files(self, mock_path):
...
-        new_file4 = self.FileAttr(dt.timestamp(dt.now()), "00001122_foo4")
-        new_file5 = self.FileAttr(dt.timestamp(dt.now()), "00001122_foo5")
-        new_file6 = self.FileAttr(dt.timestamp(dt.now()), "00001122_foo6")
-        new_file7 = self.FileAttr(dt.timestamp(dt.now()), "00001122_foo7")
-        seven_new = self.MockSFTP([new_file1, new_file2, new_file3, new_file4,
-                                    new_file5, new_file6, new_file7])
+        # When too many new files are present, AssertionError
+        file_batch = [
+            self.FileAttr(dt.timestamp(dt.now()), f"00001122_foo{i}")
+            for i in range(EXPECTED_FILES_PER_DROP + 1)
```
Comment (jingjtang) [inline changehc/tests/test_download_ftp_files.py:58]: I think there should be `for i in range(1, EXPECTED_FILES_PER_DROP + 1)`

Comment (krivard) [reply changehc/tests/test_download_ftp_files.py:58]: Consider the following: if EXPECTED_FILES_PER_DROP=7, then for this test we want to construct a list of 8 files.

```
>>> len(list(range(7+1)))
8
>>> len(list(range(1, 7+1)))
7
```

Then we should use the former.

Comment (jingjtang) [reply changehc/tests/test_download_ftp_files.py:58]: You are right. Thanks.

**Review by jingjtang — APPROVED** [review]
lgtm

**Pull request merged**